database/seeders/PlayerAchievementsSeeder.php

```diff
@@ -135,6 +138,9 @@ public function run(): void
 
             // update player count and unlock metrics
             (new UpdateGameMetricsAction())->execute($game);
+            (new UpdateGamePlayerCountAction())->execute($game);
+            (new UpdateGameBeatenMetricsAction())->execute($game);
+            (new UpdateGameAchievementsMetricsAction())->execute($game);
```

Comment:
This is related to breaking the command up over the weekend and should be committed regardless of whether or not the doc changes are approved.